release.nix

```diff
@@ -69,8 +69,11 @@ in {
   in {
   daedalus-installer = allArchesNoWindows;
   yaml2json = allArchesNoWindows;
+  nodejs = allArchesNoWindows;
```

Comment:
Let’s cache this in Hydra, since building from source takes some considerable time.